build-aux/cades-condo.cmake

```diff
@@ -0,0 +1,29 @@
+# Initial cache list for CADES-CONDO
+#
+# Usage: cmake -C /path/to/this/file /path/to/DCA/source -D<option>=<value> ...
+
+set(CMAKE_CXX_FLAGS "-fPIC -std=c++14" CACHE STRING "compiler flags")
```

Comment:
I'd prefer using add_compile_options rather than setting CMAKE_CXX_FLAGS directly.
There are often some issues with overwriting with the latter.
Also, C++14 is already set as default.

Reply:
I’d like to migrate to that as well I’ll do that